util/conan-package/conanfile.py

```diff
@@ -0,0 +1,20 @@
+import os
+from conans import ConanFile
+
+class GlmConan(ConanFile):
+    name = "glm"
+    version = "master"
+    generators = "txt"
+    url="https://github.com/g-truc/glm"
+    description="OpenGL Mathematics (GLM)"
+    license = "https://github.com/g-truc/glm/blob/manual/copying.txt"
+    exports = ["FindGLM.cmake", "lib_licenses/*", os.sep.join([".", "..", "..", "*"])]
+
+    def build(self):
+        self.output.warn("No compilation necessary for GLM")
+        self.output.warn(os.sep.join([".", "..", "..", "*"]))
```

Comment:
What is the purpose of the second warning? I would remove it, and maybe change the first one for a ``output.info()`` message, as it is not something that should concern the user.